nexus/cmd/benchmark/main.go

```diff
@@ -3,50 +3,104 @@ package main
 import (
 	"flag"
 	"fmt"
+	"runtime"
+	"sync"
 
 	"github.com/wandb/wandb/nexus/pkg/gowandb"
-	"github.com/wandb/wandb/nexus/pkg/gowandb/opts/session"
+	"github.com/wandb/wandb/nexus/pkg/gowandb/opts/sessionopts"
+	"github.com/wandb/wandb/nexus/pkg/gowandb/settings"
 )
 
-func main() {
-	host := flag.String("host", "localhost", "host to connect to")
-	port := flag.Int("port", 0, "port to connect to")
-	numHistory := flag.Int("numHistory", 1000, "number of history records to log")
-	numHistoryElements := flag.Int("numHistoryElements", 5, "number of elements in a history record")
-	teardown := flag.Bool("close", false, "flag to close the server")
-	offline := flag.Bool("offline", false, "use offline mode")
-	flag.Parse()
+type BenchOpts struct {
+	host               *string
+	port               *int
+	numHistory         *int
```

Comment:
numHistoryLines/numHistoryEntries?

Reply:
good point, ill rework this a bit in the next rev, was trying to limit the string length but it isnt obvious what it does.